src/registrar/utility/email.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for sending emails."""
+
+import boto3
+
+from django.conf import settings
+from django.template.loader import get_template
+
+
+def send_templated_email(template_name: str, to_address: str, context={}):
+    """Send an email built from a template to one email address.
+
+    template_name is relative to the same template context as Django's HTML
+    templates. context gives additional information that the template may use.
+    """
+
+    template = get_template(template_name)
+    email_body = template.render(context=context)
+
+    ses_client = boto3.client(
+        "sesv2",
+        region_name=settings.AWS_REGION,
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
```

Comment:
What will happen in environments where there are not these variables set?

Reply:
```
[14/Feb/2023 15:39:55] WARNING [registrar.models.domain_application:490] Failed to send email, no credentials are present
```

Reply:
I think that is the case - I don't think every environment needs to have these credentials.